docs/index.asciidoc

```diff
@@ -191,6 +192,16 @@ input plugins.
 
 &nbsp;
 
+[id="plugins-{type}s-{plugin}-check_archive_validity"]
+===== `check_archive_validity`
+
+* Value can be any of: `true`, `false`
+* The default is `false`.
+
+Before start read a compressed file, checks for its validity.
+This request a full read of the archive, so potentially could cost time.
+If not specified to true, and the file is corrupted, could end in cyclic processing of the broken file.
+
```

Comment:
Good explanation.  I took the information you provided, and tried to reword it a bit.  What do you think about this: 

>The `read` option kicks off a full read of an archive, and could potentially
waste time trying to process an invalid file.
When set to `true`, this option verifies that a compressed file is valid before
reading it.

>If this option is not explicitly set to `true`, a corrupt file could cause
cyclic processing of the broken file.

Reply:
If I'm not wrong, from this rewording I understand that reading an archive is costly, so before read a corrupted archive (and waste time) this option enables the verification.

In the original form I tried to describe that processing a corrupted archive led to looping on that archive and to avoid this we could enable this flag. Enabling this flag means read upfront the entire file for a verification and then read it again for processing. In this case for a not corrupted archive this could be considered a waste of time.

Reply:
What about this:

>When set to `true`, this setting verifies that a compressed file is valid before
processing it. There are two passes through the file--one pass to
verify that the file is valid, and another pass to process the file.

>Validating a compressed file requires more processing time, but can prevent a
corrupt archive from causing looping.

Reply:
Thanks @karenzone sounds really really better